project-base/storefront/pages/_app.tsx

```diff
@@ -1,12 +1,7 @@
 import 'dayjs/locale/cs';
 import 'dayjs/locale/sk';
-import 'lightgallery/css/lg-thumbnail.css';
-import 'lightgallery/css/lightgallery.css';
-import 'react-loading-skeleton/dist/skeleton.css';
 import '../styles/globals.css';
 import '../styles/user-text.css';
-import 'react-toastify/dist/ReactToastify.css';
-import 'nprogress/nprogress.css';
```

Comment:
I understand motivation behind this but this is not good step since it's huge [step back for optimalization](https://www.debugbear.com/blog/avoid-css-import). Please revert this change.

Reply:
I did not know! Thanks for pointing it out :confetti_ball: